docs/app/pages/components/components-sections/tabs/tabs/snippets/app.html

```diff
@@ -13,5 +13,5 @@ <h4>{{ tab.title }}</h4>
             </div>
         </ux-tab>
     </ux-tabset>
-
+    <button class="btn button-primary" (click)="tabset.selectTab(1)">Select Second Tab</button>
 </div>
```

Comment:
Add the class <code>button-toggle-primary</code> then there is no need for the custom css for the button

Reply:
I think a toggle button would be unnecessarily complicated, this is pretty much fine as it is. My only suggestion is to use secondary instead of primary styling.